lib/server.ts

```diff
@@ -273,9 +278,9 @@ export function createApp(devJsonPath: string, opts: DevOptions): Express.Applic
                 }
               }
             } else {
-              log.debug(`Proxying request to: ${gatewayUrl}${req.path}`);
+              log.debug(`Proxying request to: ${gateway}${req.path}`);
```

Comment:
Does this work? Isn't gateway an object? You mean gateway.bundleUrl?

Confirm by running the command with `-l DEBUG` to turn on logging